conf/messages.cy

```diff
@@ -459,6 +459,8 @@ userType.ownerTrustee = Owner or trustee for owner
 userType.occupier = Occupier’s agent
 userType.lessee = Lessee’s agent
 userType.owner = Owner’s agent
+label.userType = What is your relationship to the property?
+hint.aboutYou = If you'''re renting or leasing the property, even as a charity or non-profit organisation, you're considered the occupier. If you are both the owner and the occupier, select 'Owner or trustee for owner'.
```

Comment:
The line should use the apostrophe as in line 460

Reply:
Ah the old straight apostrophe issue! Fixed on this and all other instances!